watchtower/wtdb/client_db.go

```diff
@@ -260,6 +267,13 @@ func (c *ClientDB) CreateTower(lnAddr *lnwire.NetAddress) (*Tower, error) {
 			return ErrUninitializedDB
 		}
 
+		towerToSessionIndex := tx.ReadWriteBucket(
+			cTowerToSessionIndexBkt,
+		)
+		if towers == nil {
```

Comment:
I think this needs to be `towerToSessionIndex == nil`.

Reply:
omgggg good catch!!